src/components/QuranReader/MushafView.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
prefer to name this file something that's commonly understood. Better named as `PageView` for example. We want to avoid using Arabic words in English as not everyone spells it the same 😅

Reply:
Good called, renamed it QuranPage. Will try to avoid Arabic names moving forward insha'Allah